pg_analytics/tests/basic.rs

```diff
@@ -83,26 +83,6 @@ fn array_results(mut conn: PgConnection) {
     assert_eq!(columns[1], second);
 }
 
-#[rstest]
-fn alter(mut conn: PgConnection) {
-    "CREATE TABLE t (a int, b text) USING parquet".execute(&mut conn);
-
-    let rows: Vec<(String,)> = "SELECT column_name FROM information_schema.columns WHERE table_schema = 'public' AND table_name = 't'".fetch(&mut conn);
-    let column_names: Vec<_> = rows.into_iter().map(|r| r.0).collect();
-
-    assert_eq!(column_names, vec!["a".to_string(), "b".to_string()]);
-
-    "ALTER TABLE t ADD COLUMN c int".execute(&mut conn);
-
-    let rows: Vec<(String,)> = "SELECT column_name FROM information_schema.columns WHERE table_schema = 'public' AND table_name = 't'".fetch(&mut conn);
-    let column_names: Vec<_> = rows.into_iter().map(|r| r.0).collect();
-
-    assert_eq!(
-        column_names,
-        vec!["a".to_string(), "b".to_string(), "c".to_string()]
-    );
-}
-
```

Comment:
You can simply `#[ignore]` this test if you'd rather not remove it. It seems like it would be useful for when we re-implement `ADD COLUMN`.